src/components/AccordionForm/AccordionForm.js

```diff
@@ -87,22 +91,27 @@ export function AccordionForm(props) {
           }
           key={`accordion-form-card-${card.id}${index}`}
           ref={cardsRefs[index]}
+          aria-label={`${lang.accordionStep} ${index + 1}`}
```

Comment:
Should this not be the following?:
```typescript
aria-label={`${lang.accordionStep} ${index + 1}, ${card.title}`}
```

Reply:
Since the {card.title} inside of the legend element will be read, if we make it as above, the title will be read twice.